src/main/java/com/github/swrirobotics/config/ConfigService.java

```diff
@@ -60,12 +62,27 @@
 
     private final Logger myLogger = LoggerFactory.getLogger(ConfigService.class);
 
+    public ConfigService(ApplicationContext myAC, Environment myEnvironment) {
+        this.myAC = myAC;
+        this.myEnvironment = myEnvironment;
+    }
+
     public Configuration getConfiguration() {
         Configuration config = new Configuration();
-        URL fileUrl = null;
+
+        Set<String> profileSet = Sets.newHashSet(myEnvironment.getActiveProfiles());
+        if (profileSet.contains("test")) {
+            // Don't actually read in the config file if we're in test mode.
+            return config;
```

Comment:
Is it worth printing a warning message here? I'm wondering if we'll get someone accidentally setting a test configuration, and being really confused when this behavior is invoked without a warning message.

Reply:
Probably couldn't hurt just so that it's obvious, although it would be very hard for somebody to accidentally set the `test` profile, as they would need to add something like `-Dspring.profiles.active=test` to their command line when starting the JVM.